### PR TITLE
Change README.md to improve first time use

### PR DESCRIPTION
**Pull request by rafaelbernard**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Build (`npm run build:check` or `yarn build:check`) was run locally and any changes were pushed
- [ ] Lint (`npm run lint` or `yarn lint`) has passed locally and any fixes were made for failures

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

Update README documentation to ease first time execution.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,20 +65,66 @@ yarn audit
...
-You don't need to [Configure Gimbal](./packages/gimbal/docs/config), but we understand that defaults are _optimistic_, at least for existing projects that want to introduce performance budgeting. 
+You don't need to [Configure Gimbal](./packages/gimbal/docs/config), but we understand that defaults are _optimistic_, at least for existing projects that want to introduce performance budgeting.
 
-Here's a **sample `.gimbalrc.yml` config** file:
+To ease you be ready to use, let's start with some **sample `.gimbalrc.yml` config** files:
+
+1. Minimal
+2. Minimal with all native audits
+3. Using other audit plugins and more sample configurations
+
```
Comment (rafaelbernard) [inline README.md]: I am proposing three levels of configuration files, from a minimal to a more complex sample configuration file. Actual configuration sample leads to some potential exceptions when executing for the first time and may lead one to quit when facing the exceptions.

File: README.md [hunk 2]
```diff
@@ -65,20 +65,66 @@ yarn audit
...
+Before executing this config file you should install mentioned plugins. For instance:
+
+```sh
+# with npm
+npm install @modus/gimbal-plugin-axe @modus/gimbal-plugin-last-value @modus/gimbal-plugin-sqlite
+
+# or with yarn
+yarn add @modus/gimbal-plugin-axe @modus/gimbal-plugin-last-value @modus/gimbal-plugin-sqlite
+```
```
Comment (rafaelbernard) [inline README.md]: This was required. `-g @modus/gimbal` is not enough to run a configuration sample file that uses plugins.

File: README.md [hunk 3]
```diff
@@ -65,20 +65,66 @@ yarn audit
...
+
+In case you don't use `build` as your build directory and an exception raises with an error 
+concerning a nonexistent `build` directory, please create one and run gimbal again.
```
Comment (rafaelbernard) [inline README.md]: I am not sure yet which configuration requires this. But we have an exception when `build` directory does not exist.

Comment (grgur) [reply README.md]: We need to document all config properties someplace and note their default values. 
Also, it's worth mentioning that users need to build the project before running gimbal. That's where all the juice is

Comment (rafaelbernard) [reply README.md]: Sure. I will mention that users need to build.

**Pull request merged**